xboxone/xboxone.py

```diff
@@ -530,6 +537,10 @@ def turn_off(self):
 
     def mute_volume(self, mute):
         """Mute the volume."""
+        if self._muted == False:
```

Comment:
This doesn’t work unless HA is the only thing able to mute the TV volume. The mute command is a toggle command, which means that if you were to mute your TV via your remote, your HA state would be incorrect.

Reply:
Comparison to `True` or `False`should be done with `is`instead of `==`